chainer/gradient_check.py

```diff
@@ -412,6 +412,7 @@ def __init__(
         self.eps = eps
         self.dtype = dtype
         self.detect_nondifferentiable = detect_nondifferentiable
+        self.params_to_func = params_to_func
```

Comment:
Move this line before the comment at line 411.

Reply:
Fixed.